include/xeus-cpp/xeus_cpp_config.hpp.in

```diff
@@ -14,6 +14,9 @@
 #define XEUS_CPP_VERSION_MINOR 4
 #define XEUS_CPP_VERSION_PATCH 0
```

Comment:
We should configure the versions too. The current version should be defined in the calling cmake. Elsewhere we have it in a VERSION file which is read by cmake. @JohanMabille, what is your take here?

Reply:
Sorry I missed this comment. I think the opposite, the code should not depend on the building tool.